### DISCUSSION TITLE
Add a filter field to Preferences - Keybindings section

### DISCUSSION THREAD

**Discussion by advice2020** [geany/geany#4049] [Ideas]
Would anyone be interested in adding a filter field in the Preferences - Keybindings section to be able find actions, you are looking to set keybindings for, more easily.
Currently there is a temporary field at the bottom but only detect letters at the beginning of each action, so user would have to know the exact name of them, so this field is not really that helpful.
What I am suggesting is the exact same filter field already included in Geany in the "Plugins Manager" dialog box.
This field detects letters/words anywhere in the items name and filters the results to only show the matched keywords, which works great.  I feel this could be helpful in keybindings as well.

Thank You

**Comment by advice2020** [comment]
Recently discovered that if you hit "Ctrl+F" it brings focus to the existing search field provided by this section (the search that is only based on beginning letters when searching, which is not that helpful for finding actions to assign keybinds to)
If this feature request I mentioned above is added, would the developer who adds this also consider setting this hotkey "Ctrl+F" to bring focus to the new field added at top of the keybind section instead of the other current filter field.
Thank You

This could also go well with the following request if anyone is interested #4163

EDIT:
https://github.com/geany/geany/issues/2848
I just came across this issue/request post while doing a search for something else, yikes this was made in year 2021.
I thought this would have been a useful update, but does not look good for this feature to be added.
Hopefully this will be considered in the future, I think it could be very helpful if offered.

**Comment by eht16** [comment]
The filter entry and the Ctrl-F shortcut added in #4192. Let's continue further discussion there or in #2848.
